src/components/form/form.ts

```diff
@@ -33,26 +38,36 @@ export class MForm extends ModulVue {
     }
 
     public submit(): void {
```

Comment:
submit commence à faire pas mal d'autre chose que submitter :P faudrait faire un refactor. Genre..
`...
this.onSubmit();
this.handleErrors();
this.handleFocus();
this.handleScroll();
...
`

Mais je peux le faire dans un autre billet @Netnix vu que c'est pas ton ticket qui cause ça mais plutot une accumulation. Your call.

Reply:
En fait moi j'irais un step plus loin encore... je rendrais la gestion des erreurs totalement configurable.
Là en ce moment si qq1 décide qu'il aime pas les Toasts, il est pogné avec.

Si on avait par contre un service avec des méthodes exposés du genre
subscribe(eventType (clearFormMessages, multipleFormError, etc), callback)
clearFormMessages => On call les callback de clearFormMessages
manageMultipleFormErrors => On call les callback de multipleFormError

Ensuite dans brio on irait souscrire et configurer le comportement désiré.  Modul pourrait même exposer un comportement suggéré / par défaut via une classe dans modul (on remplacerait les callback par une classe)

Comme ça c'est modularisable / configurable

Reply:
@Mboulianne Si on veut désactiver les toasts on peut utiliser la props showErrorsInToast

Reply:
Ouais pis spreader des props partout dans notre application.. à chaque fois qu'on fait un formulaire.  Ça devient lourd.

Reply:
D'accord c'est pourquoi je préfère que les props possèdent des valeur par défault et aussi que généralement on préfère les slot aux props (composition over configuration) . 

Dans ce cas par contre, je pense qu'on veut les Toast par défault , @jipigi  peut confirmer si je me trompe mais c'est ce qui a été déterminer comme le fonctionnement UX attendu par défault.

Reply:
Oui le fonctionnement par défaut sera avec les Toasts. Au niveau UX pur, avec deux alternatives, je trouve qu'on en a déjà une de trop...

Reply:
j'y suis allé avec les modifications suggérés par @Mboulianne finalement.